lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -14,15 +14,15 @@
 // Header
 // ------
 .forum-nav-header {
-  @include box-sizing(border-box);
+  box-sizing: border-box;
   display: table;
-  border-bottom: 1px solid $gray-l2;
+  border-bottom: 1px solid $forum-color-border;
   width: 100%;
   background-color: $gray-l3;
 }
 
 .forum-nav-browse {
-    @include box-sizing(border-box);
+    box-sizing: border-box;
```

Comment:
We're using `box-sizing: border-box` a lot, are there any block elements we *don't* wan't that set for? (ie, can we do `div { b-s: bb; }`?)

I guess that's more of a review of the original structure of the stylesheet, we should just land this first and then think about cleaning it up after.